cmd/admin/cert.go

```diff
@@ -221,6 +224,23 @@ func (a *admin) serialsFromRegID(ctx context.Context, regID int64) ([]string, er
 	return serials, nil
 }
 
+func cleanSerial(serial string) (string, error) {
+	serialStrip := func(r rune) rune {
+		switch {
+		case unicode.IsLetter(r):
+			return r
+		case unicode.IsDigit(r):
+			return r
+		}
+		return rune(-1)
+	}
+	strippedSerial := strings.Map(serialStrip, serial)
+	if !core.ValidSerial(strippedSerial) {
+		return "", errors.New(strippedSerial)
```

Comment:
Returning simply `errors.New(strippedSerial)` is going to result in the error message below looking like 'skipping serial "bad!example??": badexample', which isn't necessarily super helpful. I think a descriptive error here like `fmt.Errorf("cleaned serial %q is not valid", strippedSerial)` would be clearer.